src/wasm/ExportedFunctionObject.cpp

```diff
@@ -125,7 +125,7 @@ static Value callExportedFunction(ExecutionState& state, Value thisValue, size_t
     return Value(Object::createArrayFromList(state, values));
 }
 
-ExportedFunctionObject* ExportedFunctionObject::createExportedFunction(ExecutionState& state, wasm_func_t* funcaddr)
+ExportedFunctionObject* ExportedFunctionObject::createExportedFunction(ExecutionState& state, wasm_func_t* funcaddr, size_t index)
```

Comment:
IMO you can use uint32_t instead of size_t
because String::fromDouble can be wrong when you get large number in 64 bit

Reply:
Thanks for the review. I fixed the type of index to `uint32_t`.